src/test/regress/expected/multi_extension.out

```diff
@@ -111,11 +111,15 @@ ALTER EXTENSION citus UPDATE TO '9.0-1';
 ALTER EXTENSION citus UPDATE TO '9.0-2';
 ALTER EXTENSION citus UPDATE TO '9.1-1';
 ALTER EXTENSION citus UPDATE TO '9.2-1';
+ALTER EXTENSION citus UPDATE TO '9.2-2';
+ALTER EXTENSION citus UPDATE TO '9.3-1';
```

Comment:
```suggestion
/*
 * As we mistekanly bumped version to 9.3.1 (in previous release),
 * we support updating citus from 9.3.1 to 9.2.4, but we do not
 * support explicitly updating it to to 9.3.1.
 * Hence below fail update (to 9.3.1) command should fail.
 */
ALTER EXTENSION citus UPDATE TO '9.3-1';
```

Reply:
Let's be precise about using the SQL versioning schema, which is independent of the binary versioning schema. There was never a 9.3.1, but 9.2.3 had SQL version 9.3-1.

Reply:
makes sense

Reply:
updated the suggestion

Reply:
If comment is ok for you @onderkalaci, @marcocitus & @thanodnl , I can squash and merge ?